manageability-docker/vision-docker/third-party-programs.txt

```diff
@@ -0,0 +1,1083 @@
+Intel In-Band Manageability Third Party Program Files
```

Comment:
I should have thought about this more on my original comment.  We shouldn't have duplicate files since one will get updated and not the other.  Could we link to the one at root instead?